### PR TITLE
EA Core Reading

### PR DESCRIPTION
**Pull request by jpaddison3**
Add core reading to the sidebar and sequences page, which we've renamed 'library'. Thanks @aarongertler, @oldmantax for input.

image

### CODE REVIEW

File: packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx [hunk 1]
```diff
@@ -230,26 +230,31 @@ export const menuTabs: Record<ForumTypeString,Array<MenuTab>> = {
...
-    // }, {
-    //   id: 'motivation',
-    //   title: 'Motivation',
-    //   link: '/TODO:',
-    //   tooltip: "TODO: Aaron's cool",
-    //   subItem: true,
+    }, {
+      id: 'handbook',
+      title: 'The EA Handbook',
+      link: '/handbook',
```
Comment (s-cheng) [inline packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx:243]: Is this the right link? Aaron has a comment in this post saying it's an older version: https://forum.effectivealtruism.org/handbook?commentId=MRRQYrhFemqzTSoa8

File: packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx [hunk 2]
```diff
@@ -230,26 +230,31 @@ export const menuTabs: Record<ForumTypeString,Array<MenuTab>> = {
...
+    }, {
+      id: 'replacing-guilt',
+      title: 'Replacing Guilt',
+      link: '/s/a2LBRPLhvwB83DSGq',
+      tooltip: "Nate Soares writes about replacing guilt with other feelings, exercising self-compassion, and developing confidence — so that we can create a better world.",
+      subItem: true,
+    }, {
+      id: 'most-important-century',
+      title: 'Most Important Century',
+      link: '/posts/TwQzyP3QgttmuTHym/all-possible-views-about-humanity-s-future-are-wild',
```
Comment (s-cheng) [inline packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx:255]: Did you say that we're making these posts into a proper sequence?

File: packages/lesswrong/components/collections/CollectionsCard.tsx [hunk 3]
```diff
@@ -106,7 +106,7 @@ const CollectionsCard = ({ collection, url, mergeTitle=false, classes }: {
         </Typography>
       </div>
       <div className={classes.media}>
-        <CloudinaryImage publicId={collection.imageId} />
+        <CloudinaryImage publicId={collection.imageId} width={307} height={86} />
```
Comment (s-cheng) [inline packages/lesswrong/components/collections/CollectionsCard.tsx:109]: Why did you need to add dimensions to these?

**Review by s-cheng — APPROVED** [review]
Looks good! Just had a couple questions

File: packages/lesswrong/components/collections/CollectionsCard.tsx (hunk 3 above)
Comment (jpaddison3) [reply packages/lesswrong/components/collections/CollectionsCard.tsx:109]: LessWrong apparently resized their images before uploading to cloudinary. Fortunately, cloudinary allows resizing on retrieval, so I just did that.

File: packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx (hunk 1 above)
Comment (jpaddison3) [reply packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx:243]: I think Aaron's language there is legit confusing. FWIW.

File: packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx (hunk 2 above)
Comment (jpaddison3) [reply packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx:255]: Yes, though I'm happy to launch as-is, pursuant to Aaron's signoff.
